win32ss/gdi/eng/device.c

```diff
@@ -419,21 +292,14 @@ EngpRegisterGraphicsDevice(
                   pustrDescription->Length);
     pGraphicsDevice->pwszDescription[pustrDescription->Length/sizeof(WCHAR)] = 0;
 
-    /* Initialize the pdevmodeInfo list and default index  */
+    /* Initialize the pdevmodeInfo list */
     pGraphicsDevice->pdevmodeInfo = NULL;
-    pGraphicsDevice->iDefaultMode = 0;
-    pGraphicsDevice->iCurrentMode = 0;
 
     // FIXME: initialize state flags
     pGraphicsDevice->StateFlags = 0;
 
     /* Create the mode list */
```

Comment:
At this point now it's more like "Initialize" the mode list; I suppose it's created later?

Reply:
Yes, mode list will be initialized later, in LDEVOBJ_bBuildDevmodeList